Sources/Overlays/_Testing_Foundation/Attachments/Attachment+URL.swift

```diff
@@ -51,6 +51,26 @@ extension Attachment where AttachableValue == _AttachableURLWrapper {
   ///
   /// - Throws: Any error that occurs attempting to read from `url`.
   ///
+  /// Use this initialize to create an instance of ``Attachment`` that
+  /// represents a local file or directory:
+  ///
+  /// ```swift
+  /// let url = try await FoodTruck.saveMenu(as: .pdf)
+  /// let attachment = try await Attachment(contentsOf: url)
+  /// Attachment.record(attachment)
+  /// ```
+  ///
+  /// When you call this initializer and pass it the URL of a file, it reads or
+  /// maps the contents of that file into memory. When you call this initializer
+  /// and pass it the URL of a directory, it creates a temporary ZIP file of the
```

Comment:
```suggestion
  /// and pass it the URL of a directory, it creates a temporary `zip` file of the
```

I don't think ZIP is an initialism in this context.

Reply:
Code voice also isn't correct as it's just the name of the filetype. 🤔

Reply:
https://en.wikipedia.org/wiki/ZIP_(file_format)

Historically "ZIP" is written uppercased even though it's not an initialism. Is there a style guide rule here?

Reply:
Not that I can find, no. An example, though: over where Apple document the zip file type, they use lower case. https://developer.apple.com/documentation/uniformtypeidentifiers/uttype-swift.struct/zip

Reply:
Lowercase it is.